### PR TITLE
Quick start improvements 🙉

### PR DESCRIPTION
**Pull request by roman-vynar**
I went through [Quick Start Guide](https://securitymonkey.readthedocs.io/en/latest/quickstart.html) and deployed Security 🐒 on my premises. There are some issues that I have fixed as it seems the guide is a bit outdated.

Here is the improvements to the docs, hope it will make it less painful for other users.
Thanks for the product!

### CODE REVIEW

**Comment by monkeysecurity** [comment]
Related to #594.

File: docs/quickstart.rst [hunk 1]
```diff
@@ -731,18 +733,18 @@ it were to crash.
...
     command=python /usr/local/src/security_monkey/manage.py start_scheduler
 
+Copy supervisor config::
 
-Copy /usr/local/src/security_monkey/supervisor/security_monkey.conf to /etc/supervisor/conf.d/security_monkey.conf and make sure it points to the locations where you cloned the security monkey repo.::
-
+    sudo cp /usr/local/src/security_monkey/supervisor/security_monkey.conf /etc/supervisor/conf.d/security_monkey.conf
     sudo service supervisor restart
-    sudo supervisorctl &
+    sudo supervisorctl status
```
Comment (supertom) [inline docs/quickstart.rst]: much nicer!

File: env-config/config-deploy.py [hunk 2]
```diff
@@ -52,7 +52,7 @@
     }
 }
 
-SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:securitymonkeypassword@localhost:5432/secmonkey'
+SQLALCHEMY_DATABASE_URI = 'postgresql://securitymonkeyuser:securitymonkeypassword@localhost:5432/secmonkey'
```
Comment (supertom) [inline env-config/config-deploy.py]: Thanks, I know I've been meaning to fix that one myself. :-)

File: docs/quickstart.rst [hunk 3]
```diff
@@ -371,18 +371,14 @@ Add this to /etc/hosts: (Use nano if you're not familiar with vi.)::
...
-    sudo touch /var/log/security_monkey/security_monkey.error.log
-    sudo touch /var/log/security_monkey/security_monkey.access.log
-    sudo touch /var/log/security_monkey/security_monkey-deploy.log
-    sudo chown www-data /var/log/security_monkey/security_monkey-deploy.log
 
 Let's install the tools we need for Security Monkey::
 
     $ sudo apt-get update
-    $ sudo apt-get -y install python-pip python-dev python-psycopg2 postgresql postgresql-contrib libpq-dev nginx supervisor git libffi-dev 
+    $ sudo apt-get -y install python-pip python-dev python-psycopg2 postgresql postgresql-contrib libpq-dev nginx supervisor git libffi-dev gcc
```
Comment (monkeysecurity) [inline docs/quickstart.rst]: What did you need gcc for?

Comment (roman-vynar) [inline docs/quickstart.rst]:
```
Processing dependencies for security-monkey==0.8.0
Searching for cryptography==1.7.1
Reading https://pypi.python.org/simple/cryptography/
Best match: cryptography 1.7.1
Downloading https://pypi.python.org/packages/82/f7/d6dfd7595910a20a563a83a762bf79a253c4df71759c3b228accb3d7e5e4/cryptography-1.7.1.tar.gz#md5=2f63dee6ace602d1d20d1995f8538015
Processing cryptography-1.7.1.tar.gz
Writing /tmp/easy_install-D6AxH5/cryptography-1.7.1/setup.cfg
Running cryptography-1.7.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-D6AxH5/cryptography-1.7.1/egg-dist-tmp-YXkiOj
unable to execute 'x86_64-linux-gnu-gcc': No such file or directory
unable to execute 'x86_64-linux-gnu-gcc': No such file or directory

    No working compiler found, or bogus compiler options
    passed to the compiler from Python's distutils module.
    See the error messages above.
    (If they are about -mno-fused-madd and you are on OS/X 10.8,
    see http://stackoverflow.com/questions/22313407/ .)
error: Setup script exited with 1
```

Comment (monkeysecurity) [inline docs/quickstart.rst]: fair enough :)

**Pull request merged**